suite-native/module-check-backup/src/hooks/useNavigateCheckBackup.ts

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { selectDeviceModel } from '@suite-common/wallet-core';
+import {
+    DeviceCheckBackupStackRoutes,
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { DeviceModelInternal, models } from '@trezor/device-utils';
+
+const checkBackupUnsupportedDeviceModels: DeviceModelInternal[] = [DeviceModelInternal.T1B1];
+
+type NavigationProp = StackNavigationProps<
+    DeviceSettingsStackParamList,
+    DeviceSettingsStackRoutes.DeviceSettings | DeviceSettingsStackRoutes.ConfirmFirmwareUpdate
+>;
+
+export const useNavigateCheckBackup = () => {
```

Comment:
nit: 
```suggestion
export const useNavigateToCheckBackup = () => {
```

So the return value is equal to the hook name.

Reply:
https://github.com/trezor/trezor-suite/pull/20014/commits/b2cfded2d94c2c19c28eb6d02338fec42508772f